libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -544,76 +544,76 @@ void AC_Avoid::adjust_roll_pitch_rad(float &roll_rad, float &pitch_rad, float ve
 
 /*
  * Note: This method is used to limit velocity horizontally only 
- * Limits the component of desired_vel_cms in the direction of the unit vector
+ * Limits the component of desired_vel in the direction of the unit vector
  * limit_direction to be at most the maximum speed permitted by the limit_distance_cm.
  *
  * Uses velocity adjustment idea from Randy's second email on this thread:
  * https://groups.google.com/forum/#!searchin/drones-discuss/obstacle/drones-discuss/QwUXz__WuqY/qo3G8iTLSJAJ
  */
-void AC_Avoid::limit_velocity_2D(float kP, float accel_cmss, Vector2f &desired_vel_cms, const Vector2f& limit_direction, float limit_distance_cm, float dt)
+void AC_Avoid::limit_velocity_2D(float kP, float accel, Vector2f &desired_vel, const Vector2f& limit_direction, float limit_distance, float dt) const
```

Comment:
Could we put a comment on this method indicating that the units you get out are the units your put in, perhaps - but it is on the caller to make sure they are consistent?